docroot/modules/custom/mass_entity_usage/tests/src/ExistingSite/EntityUsageTest.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+namespace Drupal\Tests\mass_entity_usage\ExistingSite;
+
+use Drupal\file\Entity\File;
+use Drupal\mass_content_moderation\MassModeration;
+use Drupal\node\Entity\Node;
+use Drupal\user\Entity\User;
+use Drush\Drupal\Commands\core\QueueCommands;
+use Drush\Log\DrushLoggerManager;
+use weitzman\DrupalTestTraits\Entity\MediaCreationTrait;
+use weitzman\DrupalTestTraits\ExistingSiteBase;
+use weitzman\LoginTrait\LoginTrait;
+
+/**
+ * Class EntityUsageTest.
+ */
+class EntityUsageTest extends ExistingSiteBase {
+
+  use LoginTrait;
+  use MediaCreationTrait;
```

Comment:
Lets use https://github.com/drupaltest/queue-runner-trait/. That package is already in our project. Should simplify our test.

Reply:
Awesome! I didn't know about that trait. Thanks.

Reply:
The suggestion is/was implemented. Thanks.